.jshintrc

```diff
@@ -16,7 +16,7 @@
     "immed"           : true,      // Prohibits the use of immediate function invocations without wrapping them in parentheses.
     // "indent"       : 4,         // Enforces specific tab width for your code. Has no effect when "white" option is not used.
     "latedef"         : "nofunc",  // Prohibits the use of a variable before it was defined. Setting this option to "nofunc" will allow function declarations to be ignored.
-    "newcap"          : true,      // Requires you to capitalize names of constructor functions.
+    "newcap"          : false,     // Requires you to capitalize names of constructor functions.
```

Comment:
Reviewers: This property is deprecated in JSHint now so I've decided to just disable it:

http://jshint.com/docs/options/#newcap